### PR TITLE
Initial commit with actions and logos added

### PR DESCRIPTION
**Pull request by cs-adarsh-gupta**
#### UTCs: 
 
- [x] Connector installation verified. 
- [x] Connector logo verified. 
- [x] Actions and Playbook list verified. 
- [x] Playbook tags verified. 
- [x] All Playbook are in info mode verified. 
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: taegis-xdr/info.json [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+  "configuration": {
+    "fields": [
+      {
+        "name": "client_id",
+        "title": "Client ID",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "tooltip": "Specify the Client ID as described in the Taegis Documentation.",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json]: provide the doc link here

File: taegis-xdr/info.json [hunk 2]
```diff
@@ -0,0 +1,702 @@
...
+        "description": "Specify the Client ID as described in the Taegis Documentation."
+      },
+      {
+        "name": "client_secret",
+        "title": "Client Secret",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "tooltip": "Specify the Client Secret as described in the Taegis Documentation",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json]: provide the doc link here

File: taegis-xdr/info.json [hunk 3]
```diff
@@ -0,0 +1,702 @@
...
+      {
+        "name": "environment",
+        "title": "Environment",
+        "type": "select",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "tooltip": "Specify the Environment to utilize Secureworks Taegis XDR. Server URL changes with change in Environment.",
+        "description": "Specify the Environment to utilize Secureworks Taegis XDR. Server URL changes with change in Environment.",
+        "options": [
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json:44]: Make the changes in the options for a better UI experience
Like US - 1, US - West

File: taegis-xdr/info.json [hunk 4]
```diff
@@ -0,0 +1,702 @@
...
+          "required": false,
+          "tooltip": "Specify the ordering of the AllAssets and SearchAssets queries. By Default, It Orders by hostname.",
+          "description": "Specify the ordering of the AllAssets and SearchAssets queries. By Default, It Orders by hostname."
+        },
+        {
+          "name": "order_direction",
+          "title": "Order Direction",
+          "type": "select",
+          "value": "asc",
+          "options": "asc,desc",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json]: Change the options to Ascending or Decesding

File: taegis-xdr/info.json [hunk 5]
```diff
@@ -0,0 +1,702 @@
...
+          "required": false,
+          "tooltip": "Specify the number of investigations to be shown per page. Maximum value can be 100",
+          "description": "Specify the number of investigations to be shown per page. Maximum value can be 100"
+        },
+        {
+          "name": "order_by_field",
+          "title": "Order by Field",
+          "type": "select",
+          "value": "id",
+          "options": "id,tenant_id,tags,genesis_alerts,genesis_events,alerts,events,assets,auth_credentials,key_findings,description,created_at,updated_at,notified_at,created_by,status,contributors,service_desk_id,service_desk_type,all_alerts,all_events,priority,type",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json]: Change the options

File: taegis-xdr/info.json [hunk 6]
```diff
@@ -0,0 +1,702 @@
...
+          "required": false,
+          "tooltip": "Specify the ordering of the AllInvestigations queries. By Default, It Orders by ID.",
+          "description": "Specify the ordering of the AllInvestigations queries. By Default, It Orders by ID."
+        },
+        {
+          "name": "order_direction",
+          "title": "Order Direction",
+          "type": "select",
+          "value": "asc",
+          "options": "asc,desc",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json]: Change the options

File: taegis-xdr/info.json [hunk 7]
```diff
@@ -0,0 +1,702 @@
...
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the IDs of the Alerts in comma separated manner whose resolution status you want to update.",
+          "description": "Specify the IDs of the Alerts in comma separated manner whose resolution status you want to update."
+        },
+        {
+          "name": "resolution_status",
+          "title": "Resolution Status",
+          "type": "select",
+          "options": "OPEN,TRUE_POSITIVE_BENIGN,TRUE_POSITIVE_MALICIOUS,FALSE_POSITIVE,NOT_ACTIONABLE,OTHER,SUPPRESSED",
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/info.json]: Change options

File: taegis-xdr/operations.py [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
...mail_compromise {{ source_address_geo_summary {{ city {{ confidence name locale_names {{ record {{ key value }} }} geoname_id }} location {{ radius metro_code timezone longitude us_metro_code gmt_offset latitude }} asn {{ autonomous_system_org autonomous_system_no }} country {{ confidence code iso_code geoname_id }} continent {{ code geoname_id }} }} user_name source_address }} }} priority {{ version model_version model_name applied_time {{ seconds nanos }} value prioritizer evidence }} sensor_types events_metadata {{ began_at {{ seconds nanos }} first_event_id last_event_id updated_at {{ seconds nanos }} total_events ended_at {{ seconds nanos }} }} reference_details {{ reference {{ description url type }} }} group_key entities {{ relationships {{ to_entity relationship from_entity type }} entities }} event_ids {{ id }} tags suppressed resolution_history {{ user_id timestamp {{ seconds nanos }} id num_alerts_affected reason status }} tenant_id key_entities {{ entity label }} observation_ids {{ id }} visibility suppression_rules {{ id version }} alerting_rules {{ id version }} collection_ids {{ id }} status attack_technique_ids investigation_ids {{ id GenesisAlertsFlag }} }} part }}
```
Comment (cs-utkarsh-surana) [inline taegis-xdr/operations.py]: Can you put all the static code in a constant file

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: taegis-xdr/info.json (hunk 1 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json]: Done

File: taegis-xdr/info.json (hunk 2 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json]: Done

File: taegis-xdr/info.json (hunk 3 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json:44]: Done

File: taegis-xdr/info.json (hunk 4 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json]: Done

File: taegis-xdr/info.json (hunk 5 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json]: Done

File: taegis-xdr/info.json (hunk 6 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json]: Done

File: taegis-xdr/info.json (hunk 7 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/info.json]: Done

File: taegis-xdr/operations.py (hunk 8 above)
Comment (cs-adarsh-gupta) [reply taegis-xdr/operations.py]: Done

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**